### PR TITLE
removed incorrect line related to removing synced members

### PR DESCRIPTION
**Pull request by thefactremains**

### CODE REVIEW

File: source/deployment/ldap-group-sync.rst [hunk 1]
```diff
@@ -137,7 +137,7 @@ Users who have logged in and accessed Mattermost will be visible in the members
 
 .. image:: ../images/Group_Members.png
 
-Users can be removed from the Mattermost group on subsequent synchronizations. However, they will not be removed from teams and channels.
+Users can be removed from the Mattermost group on subsequent synchronizations.
```
Comment (mkraft) [inline source/deployment/ldap-group-sync.rst:140]: Alternatively we could say "However, they will not be removed from teams and channels unless the team or channel is group-synced.

Comment (wiersgallak) [reply source/deployment/ldap-group-sync.rst]: @thefactremains I like Martin's edit as its a bit more explicit.

File: source/deployment/ldap-group-sync.rst [hunk 2]
```diff
@@ -137,7 +137,7 @@ Users who have logged in and accessed Mattermost will be visible in the members
 
 .. image:: ../images/Group_Members.png
 
-Users can be removed from the Mattermost group on subsequent synchronizations. However, they will not be removed from teams and channels.
+Users can be removed from the Mattermost group on subsequent synchronizations. However, they will not be removed from teams and channels unless the team or channel is group-synced
```
Comment (mkraft) [inline source/deployment/ldap-group-sync.rst:140]:
```suggestion
Users can be removed from the Mattermost group on subsequent synchronizations. However, they will not be removed from teams and channels unless the team or channel is group-synced.
```

Comment (mkraft) [reply source/deployment/ldap-group-sync.rst:140]: Looks like the period was removed by accident.

**Review by wiersgallak — APPROVED** [review]
LGTM.

**Pull request merged**